examples/todoApp/src/todoApp.wasp

```diff
@@ -9,8 +9,8 @@ app todoApp {
     title: "ToDo App"
 }
 
+route "/" -> page Main
```

Comment:
Maybe don't bother with one for now? Because this is related to the example that is hosted on the webpage as demo, so maybe it is better to update this as a batch thing once we are ready to also deploy it.

Reply:
ok makes sense